init.lua

```diff
@@ -5,14 +5,39 @@ local MODPATH = minetest.get_modpath("lavastuff")
 local COOLDOWN = dofile(MODPATH.."/cooldowns.lua")
 local S
 
-if minetest.get_translator ~= nil then
-	S = minetest.get_translator(minetest.get_current_modname())
-else
-	S = function(str)
-		return(str)
-	end
-end
-
+if minetest.get_translator ~= nil then
+	S = minetest.get_translator("mobs")
```

Comment:
Why "mobs"? Was this a conversion accident?

If you want instead of the suggested change you could add a `MODNAME` variable above `MODPATH` and use that in the place of `"lavastuff"`

```suggestion
	S = minetest.get_translator("lavastuff")
```

Reply:
solved in commit [9cac4ad](https://github.com/minetest-mods/lavastuff/pull/16/commits/9cac4ad93195a50ab2a1a0c4292d0ac329bef730) using core global var by minetest global variable from api